policy/approval/predicate.go

```diff
@@ -28,6 +28,7 @@ type Predicates struct {
 	AuthorIsOnlyContributor *predicate.AuthorIsOnlyContributor `yaml:"author_is_only_contributor"`
 
 	TargetsBranch *predicate.TargetsBranch `yaml:"targets_branch"`
+	SourceBranch  *predicate.SourceBranch  `yaml:"source_branch"`
```

Comment:
I think calling this `from_branch` would match the other names a bit better as a predicate: "if targets branch", "if has author in", "if from branch", etc

You'll also need to add an `if` statement around L65 to append this to the predicate list when it is set.

Reply:
Yeah, that works for me. I switched the option and predicate name over, but still used "source branch" to refer to the branch itself since sentences like "From branch did not match..." felt a little clumsy.

Also added the predicate to the list. Thanks for that.